action_name: /right_arm_controller/follow_joint_trajectory

joints:
        - arm_right_1_joint
        - arm_right_2_joint
        - arm_right_3_joint
        - arm_right_4_joint
        - arm_right_5_joint
        - arm_right_6_joint
        - arm_right_7_joint

trans_time: 1.0

time_slowing_factor: 1.0

way_points:
        [
[ 0.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ -0.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.8, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.8, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.9, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.9, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.8, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.8, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.9, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 1.9, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.1, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.2, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.3, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.4, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.5, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.6, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.7, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.8, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.8, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.9, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 2.9, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 3.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 3.0, 1.2, 0.0, 0.0, 0.0, 0.0, 0.0]
        ]

way_point_velocities:
        [
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
[ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

        ]
